packages/ComponentLibrary/src/components/TextInput/TextInput.styles.ts

```diff
@@ -0,0 +1,128 @@
+import { CSSProperties } from 'react';
```

Comment:
To make it more readable, wouldn't it be better to group the styles by CSSProperties and SxProps in one constant each?

To export fewer files and to make importing them less cumbersome.